src/v/cloud_storage/remote_segment.h

```diff
@@ -35,6 +35,25 @@
 
 namespace cloud_storage {
 
+<<<<<<< HEAD
+=======
```

Comment:
Leftover rebase artefacts

Reply:
Weird, I just used vs-code interface to resolve (clicked keep both changes) and saved.